std_msgs/Header header
geometry_msgs/Pose pose
geometry_msgs/Pose velocity
geometry_msgs/Pose acceleration
int[3] interaction
int[3] compensation